docs/uml/activity_diagram.txt

```diff
@@ -46,6 +46,8 @@ Controller -> Test: throttle, steering
 Test -> MockPlant: command(throttle, steering)
 Test -> MockPlant: getState()
 MockPlant -> Test: speed, heading
+MockPlant -> Test: getWheelRPM()
```

Comment:
I think this line is unnecessary; we're not using the resulting RPM value (that would be set prior to all this activity).

Reply:
I think we still need this calculation for the Phase 0 requirement to "output...the two drive wheel velocities". If we command a wheel RPM based on a centerline wheel, the actual left/right wheels will be +/- some RPM based on the different turning circle radii. Not an issue with a bicycle, but with a 4-wheel rover with individually driven wheels will need these corrections.

(EG: Rover with a steering angle sufficient to pivot around the right rear wheel. RR RPM will be 0, LR RPM will be some value, average of two would be centerline wheel RPM -> robot speed.)

Reply:
Hmm; I gotcha; I forgot about that requirement. In that case shouldn't it be:
```
Test -> Controller: getWheelRPM()
Controller -> Test: w_left, w_right
```
Or something like that? The `MockPlant` is just for our internal tests.

Reply:
Sounds good to me, I can update the txt and png here shortly.